inc/saiswitch.h

```diff
@@ -3048,6 +3048,33 @@ typedef enum _sai_switch_attr_t
      */
     SAI_SWITCH_ATTR_MAX_ICMP_ECHO_SESSION,
 
+    /**
+     * @brief Maximum size of the packet after trimming
+     *
+     * @type sai_uint32_t
+     * @flags CREATE_AND_SET
+     * @default 128
+     */
+    SAI_SWITCH_ATTR_PACKET_TRIMMING_SIZE,
+
+    /**
+     * @brief New packet trimming DSCP value
+     *
+     * @type sai_uint8_t
+     * @flags CREATE_AND_SET
+     * @default 0
+     */
+    SAI_SWITCH_ATTR_PACKET_TRIMMING_DSCP_VALUE,
```

Comment:
Should DSCP and QUEUE_INDEX be on a more granular level like on every buffer profile or port?

Reply:
It can be, but the initial proposal is to have a global setting, which is enough for now.